css/css.go

```diff
@@ -1201,6 +1201,35 @@ func (c *cssMinifier) minifyProperty(prop Hash, values []Token) []Token {
 			values[0].Data = oneBytes
 			values[0].Ident = 0
 		}
+	case UnicodeRange:
+		for i, value := range values {
```

Comment:
Be aware that `value` may be just a CommaToken which you may need to `continue`?